app.py

```diff
@@ -1,74 +1,127 @@
 #!/usr/bin/env python3
 from flask import *
 from requests import post
-from json import load, dump
 from re import sub
 
 with open("config.json") as f:
     config = json.load(f)
 
+assert "secret" in config, "Gitlab secret token in config not defined"
+assert "token" in config, "Discord token in config not defined"
```

Comment:
enabled_hooks check should be done up here instead of returning
In addition, why have the enabled_hooks check if gitlab has checkboxes to pick what kind of webhooks are sent to endpoints?